httpx/utils.py

```diff
@@ -107,6 +107,16 @@ def get_netrc_login(host: str) -> typing.Optional[typing.Tuple[str, str, str]]:
     return netrc_info.authenticators(host)  # type: ignore
 
 
+def get_ca_bundle_from_env() -> typing.Optional[str]:
+    ssl_path = Path(os.getenv("SSL_CERT_FILE", ""))
+    if ssl_path and ssl_path.is_file():
+        return str(ssl_path)
+    ssl_path = Path(os.getenv("SSL_CERT_DIR", ""))
+    if ssl_path and ssl_path.is_dir():
```

Comment:
I don't think this is doing what we want: `bool(ssl_path)` is always going to be `True` because it refers to a `Path` object.

As a result, we'll probably end up using the current directory as the `SSL_CERT_DIR` in all cases. 😕 

We should probably instead write something like…

```python
if "SSL_CERT_DIR" in os.environ:
    ssl_path = Path(os.environ["SSL_CERT_DIR"])
    if ssl_path.is_dir():
        return str(ssl_path)
```

and do the same kind of procesing for `SSL_CERT_FILE` above (although the bug wouldn't happen because the current directory is never a file).

Reply:
(Trivia: I think this is typically a case where the Python 3.8 walrus operator would shine 😄)

```python
if (
    ssl_cert_dir := os.getenv("SSL_CERT_DIR") is not None
    and (ssl_path := Path(ssl_cert_dir)).is_dir()
):
    return str(ssl_path)
```